bin/summarize_results.py

```diff
@@ -222,7 +224,13 @@ def long2wide(df: pd.DataFrame, peptide_col: str) -> pd.DataFrame:
         # Identify non-predictor columns to keep as index
         meta_columns = [col for col in df.columns if not any([x in col for x in ['predictor', 'allele', 'BA', 'rank', 'binder']])]
         # In some rare cases meta columns can be misinterpreted as str instead of integer
-        df[meta_columns] = df[meta_columns].apply(lambda col: col.map(lambda x: pd.to_numeric(x, errors='ignore')))
+        def try_numeric(x):
```

Comment:
Whats the diff of this function compared to `pd.to_numeric(x, errors='ignore')`?